docs/decisions/.markdownlint.yml

```diff
@@ -0,0 +1,19 @@
+default: true
```

Comment:
I am still getting familiar with the codebase. Is this markdown yml file for this [markdownlint](https://github.com/DavidAnson/markdownlint) repo?